core/model/job/step/gotolocation/gotolocation.go

```diff
@@ -0,0 +1,105 @@
+package gotolocation
+
+import (
+	"context"
+	"time"
+)
+
+type State int
+type Event int
+
+const (
+	New State = iota
+	Moving
+	Near
+	Arrived
+	Cancelled
+)
+
+func (s State) String() string {
+	return [...]string{"New", "Moving", "Near", "Arrived", "Cancelled"}[s]
+}
+
+const (
+	MoveEvent Event = iota
+	NearEvent
+	ArrivedEvent
+	CancelledEvent
+)
+
+func (e Event) String() string {
+	return [...]string{"MoveEvent", "NearEvent", "ArrivedEvent", "CancelledEvent"}[e]
+}
+
+type GoToLocation struct {
+	DriverID                string  `json:",string"`
+	DestinationLatitude     float64 `json:",string"`
+	DestinationLongitude    float64 `json:",string"`
+	LastModifiedAt          string  `json:",string"`
```

Comment:
`time.Time`?

Reply:
yes the UpdatedAt (i think i will rename it) time in RFC3339 format (2006-01-02T15:04:05Z07:00). 
we can to millis timestamp. string makes it human readable, and in this particular case it's for pure informational purpose.